code/ui/GameTimer.cs

```diff
@@ -0,0 +1,23 @@
+using Sandbox.UI;
+using Sandbox.UI.Construct;
+
+namespace TTTReborn.UI
+{
+    public class GameTimer : Panel
+    {
+        public Label TimeLabel { set; get; }
+
+        public GameTimer()
+        {
+            StyleSheet.Load("/ui/GameTimer.scss");
+
+            TimeLabel = Add.Label("00:00", "timelabel");
```

Comment:
Not sure if the team discussed, should we have some system of adding `TODO`? Since this needs to be hooked up later.

Reply:
https://trello.com/b/Q5xOoF4Y/ttt-reborn

Reply:
ah right, neat.